modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -193,3 +210,64 @@ test('create a notification using role type', async ({
+	} as WorkflowDefinition;
+
+	const workflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+			workflowDefinitionName,
+			workflowDefinitionContent
+		);
+
+	workflowDefinitionIds.push(workflowDefinition.id);
+
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.clickNode('Task');
+
+	const notificationEntry = processBuilderPage.page.getByRole('link', {
+		name: 'notification 1',
+	});
+
+	await expect(notificationEntry).toBeVisible();
+
+	await notificationEntry.click();
+
+	await expect(
+		page.getByRole('tablist').filter({hasText: 'Recipient Type'})
+	).toHaveCount(6);
+
+	await diagramViewPage.clickSourceViewButton();
+
+	await page.locator('pre').nth(1).click();
+
+	await page.keyboard.press('Control+A');
+
+	await page.keyboard.press('Control+C');
+
+	const clipboardContent = await page.evaluate(() =>
+		navigator.clipboard.readText()
+	);
+
+	expect(
+		await countSubstringOccurrences(clipboardContent, `receptionType="bcc"`)
```

Comment:
Could you check if we can check for `<recipients receptionType="bcc">`

Reply:
yes, works the same 😄